docs/how-to-guides/deployment-options.md

```diff
@@ -38,6 +38,66 @@ To learn more about the various options available to customize your service, and
 
 #### Option 2: Deploy a single Bicep file locally
 
+Deploy the [Single Deploy Bicep Template](../deploy/FhirConverter-SingleAzureDeploy.bicep) by running the following command:
+
+```
+az deployment sub create --location <Location> --template-file FhirConverter-SingleAzureDeploy.bicep
+```
+
+Note: See [region availability](https://azure.microsoft.com/en-us/explore/global-infrastructure/products-by-region/?products=monitor,storage,container-apps) for the required resources to select a valid location for the resources to be deployed in.
+
+You will need to provide a *serviceName* that will be used to generate a name for each of the resources provisioned, and the *containerAppImageTag*, which is the tag of the FHIR Converter image version to be pulled from MCR. To see available image tags, visit the [FHIR Converter MCR page](https://mcr.microsoft.com/en-us/product/healthcareapis/fhir-converter/tags).
+
```

Comment:
This is fine for now, but let's consider creating a tabular format for parameters.